Bio/Sequencing/Ace.py

```diff
@@ -286,219 +286,222 @@ def __init__(self, line=None):
             self.uorc = header[5]
 
 
-def parse(source):
-    """Iterate of ACE file contig by contig.
+def _parse(handle):
```

Comment:
Can you give this private function a docstring, ideally briefly explain why it exists separately from the public parse function?

Reply:
done